bittrex_websocket/websocket_client.py

```diff
@@ -195,6 +195,7 @@ def disconnect(self):
     async def _is_query_invoke(self, kwargs):
         if 'R' in kwargs and type(kwargs['R']) is not bool:
             invoke = self.invokes[int(kwargs['I'])]['invoke']
+            ticker = self.invokes[int(kwargs['I'])].get('ticker')
```

Comment:
Should be omitted from here because private methods also pass through `_is_query_invoke` and this would put the api keys in the payload.

Should replace line 207, i.e `msg['ticker']=self.invokes[int(kwargs['I'])].get('ticker')`